src/cli-access-protected-api/cli.py

```diff
@@ -0,0 +1,54 @@
+"""
+This application represents a confidential client application that is going to be
+calling Microsoft Graph as itself. This is a confidential client in that the
+application will need its own identity and security token to perform the work, and
+it is not doing this on behalf of an enterprise user.
+
+An example of this might be a nightly script that runs a query against a database,
+and based on the results sends an email to multiple people in the organization.
+"""
+import sys
+import requests
+import json
+
+from click import command, option
+from msal import ConfidentialClientApplication
+
+@command()
+@option('-t', '--tenant', type=str, required=True, prompt='Enter the tenent name or id (GUID) in which this application was regsitered. E.g. contoso.onmicrosoft.com', help='Either the "Primary domain" or "Tenant ID" for the Azure Active Directory instance this application was registered in. E.g. contoso.onmicrosoft.com')
```

Comment:
Some general rules to follow:

- Capitalize all acronyms (id > ID)
- Run all comments through a spellchecker (tenant, registered)
- Use exact naming of portal UI elements when referring to them
- Use "Example:" instead of "E.g." or "e.g."

Reply:
Thanks.  Here's what's changed in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/2/commits/3729dd95d9069faad8191d156b82d71b7fcfffec

* acronyms have been capitalized
* spell checker ran, whoops!
* updated all references to portal elements to use their exact names and casing
* Moved to "Example" instead of E.g.